samples/shields/lmp90100_evb/rtd/src/main.c

```diff
@@ -60,7 +60,7 @@ static double rtd_temperature(int nom, double resistance)
 
 void main(void)
 {
-	const struct device *lmp90100;
+	const struct device *lmp90100 = DEVICE_DT_GET_ONE(ti_lmp90100);
```

Comment:
Just a quick question: what should samples use now, `DEVICE_DT_GET_ONE` or `DEVICE_DT_GET_ANY`? It may be confusing for new users to have both.

Reply:
Sigh. I was hoping nobody would ask that.

I don't know if there is a best answer. DEVICE_DT_GET_ANY gives better error messages, but they're at runtime. DEVICE_DT_GET_ONE fails fast, but the error message when there's no such device is very confusing to beginners.